plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/json/Image.java

```diff
@@ -10,31 +10,33 @@
  *******************************************************************************/
 package org.eclipse.che.plugin.docker.client.json;
 
+import java.util.Arrays;
+import java.util.Map;
+
 /** @author andrew00x */
 public class Image {
-    private String repository;
-    private String tag;
-    private String id;
-    private long   created;
-    private long   size;
-    private long   virtualSize;
-
-    public String getRepository() {
-        return repository;
-    }
-
-    public void setRepository(String repository) {
-        this.repository = repository;
-    }
+    private String[]            repoTags;
+    private String              parentId;
+    private String              id;
+    private long                created;
+    private long                size;
+    private long                virtualSize;
+    private Map<String, String> labels;
 
-    public String getTag() {
-        return tag;
+    /**
+     * Return a collection of tags grouped under a common prefix (the name component before :).
+     */
+    public String[] getRepoTags() {
+        return repoTags;
     }
 
-    public void setTag(String tag) {
-        this.tag = tag;
+    public void setRepoTags(String[] repoTags) {
+        this.repoTags = repoTags;
     }
 
+    /**
+     * Return randomly generated, 256-bit, hexadecimal encoded uniquely identifies the image.
```

Comment:
Describe is as docker image id. Current explanation is too precise as for client of Docker API